site/docs/components/date-picker.mdx

```diff
@@ -12,9 +12,9 @@ needToKnow:
 headerImage: date-picker
 health:
   designed: true
-  documented: true
-  implemented: false
-  latestDesign: false
+  documented: false
```

Comment:
I'll add a card to update the documentation below as it's not entirely fitting now we have range as well.